### PR TITLE
feat: allow global org users to create org and self delete

### PR DESCRIPTION
**Pull request by livio-a**
- adds a new RemoveMyUser to the AuthAPI and to the Personal Information in Console
- adds a new role `SELF_MANAGEMENT_GLOBAL` able to create an org, read policy and delete its own user
- setup step 21 changes the role `ORG_PROJECT_CREATOR` to `SELF_MANAGEMENT_GLOBAL` on global org members

### CODE REVIEW

File: console/src/assets/i18n/de.json [hunk 1]
```diff
@@ -182,15 +182,21 @@
       "STATE": "Status",
       "DELETE": "Benutzer löschen",
       "UNLOCK": "Benutzer entsperren",
-      "LOCKEDDESCRIPTION": "Dieser Benutzer wurde aufgrund der Überschreitung der maximalen Anmeldeversuche gesperrt und muss zur erneuten Verwendung entsperrt werden."
+      "LOCKEDDESCRIPTION": "Dieser Benutzer wurde aufgrund der Überschreitung der maximalen Anmeldeversuche gesperrt und muss zur erneuten Verwendung entsperrt werden.",
+      "DELETEACCOUNT": "Account löschen",
+      "DELETEACCOUNT_DESC": "Hier können Sie Ihr persönliches Konto mitsamt seinem Inhalt löschen. Diese Aktion kann nicht rückgängig gemacht werden.",
```
Comment (fgerschwiler) [inline console/src/assets/i18n/de.json:187]:
```suggestion
      "DELETEACCOUNT_DESC": "Wenn du diese Aktion ausführst, wirst du abgemeldet und danach keinen Zugriff mehr auf dein Konto haben. Diese Aktion kann nicht rückgängig gemacht werden.",
```

File: console/src/assets/i18n/en.json [hunk 2]
```diff
@@ -182,15 +182,21 @@
       "STATE": "Status",
       "DELETE": "Delete User",
       "UNLOCK": "Unlock User",
-      "LOCKEDDESCRIPTION": "This user has been locked out due to exceeding the maximum login attempts and must be unlocked to be used again."
+      "LOCKEDDESCRIPTION": "This user has been locked out due to exceeding the maximum login attempts and must be unlocked to be used again.",
+      "DELETEACCOUNT": "Delete Account",
+      "DELETEACCOUNT_DESC": "Permanently remove your Personal Account and all of its contents. This action is not reversible, so please continue with caution.",
```
Comment (fgerschwiler) [inline console/src/assets/i18n/en.json:187]:
```suggestion
      "DELETEACCOUNT_DESC": "If you perform this action, you will be logged out and will no longer have access to your account. This action is not reversible, so please continue with caution.",
```

File: console/src/assets/i18n/it.json [hunk 3]
```diff
@@ -182,15 +182,21 @@
       "STATE": "Stato",
       "DELETE": "Elimina utente",
       "UNLOCK": "Sblocca utente",
-      "LOCKEDDESCRIPTION": "Questo utente \u00e8 stato bloccato a causa del superamento dei tentativi massimi di accesso e deve essere sbloccato per essere utilizzato di nuovo."
+      "LOCKEDDESCRIPTION": "Questo utente \u00e8 stato bloccato a causa del superamento dei tentativi massimi di accesso e deve essere sbloccato per essere utilizzato di nuovo.",
+      "DELETEACCOUNT": "Elimina account personale",
+      "DELETEACCOUNT_DESC": "Rimuovi il tuo Account Personale con tutti i suoi dati. Questa azione non è reversibile, quindi continua con cautela.",
```
Comment (fgerschwiler) [inline console/src/assets/i18n/it.json:187]:
```suggestion
      "DELETEACCOUNT_DESC": "Se esegui questa azione, sarai disconnesso e non avrai più accesso al tuo account. Questa azione non può essere invertita.",
```

**Review by fgerschwiler — COMMENTED** [review]
I think somewhere in auth we need to change the role we give to users registering in global?

**Comment by livio-a** [comment]
> I think somewhere in auth we need to change the role we give to users registering in global?

you're totally right... fixed it

**Review by fgerschwiler — DISMISSED** [review]

**Review by adlerhurst — APPROVED** [review]

**Pull request merged**